src-tauri/src/northstar/install.rs

```diff
@@ -96,6 +96,22 @@ async fn do_install(
     log::info!("Extracting Northstar...");
     extract(nfile, std::path::Path::new(&extract_directory))?;
 
+    if game_install.profile != "R2Northstar" {
```

Comment:
I'd stick the `R2Northstar` string into a constant in `constants.rs` to avoid hardcoding the string in multiple places ^^